src/javascript/app_2/App/Components/Elements/account_switcher.jsx

```diff
@@ -14,12 +16,11 @@ const getAccountInfo = (loginid) => {
         loginid,
         is_virtual,
         icon : account_type.toLowerCase(), // TODO: display the icon
-        title: localize('[_1] Account', [account_type]),
+        title: account_type.toLowerCase() === 'virtual' ? localize('demo') : account_type,
```

Comment:
Can we localize the exact word we're going to display (`DEMO`)? Because using CSS to capitalize text will result in inconsistency and issues in different languages.